tao_compiler/mlir/disc/BUILD

```diff
@@ -1,5 +1,5 @@
 load(
-    "//tensorflow:tensorflow.bzl",
+    "@org_tensorflow//tensorflow:tensorflow.bzl",
```

Comment:
nice, this works on both side? `tao_compiler` and `tf_community`.

Reply:
Yes, in `tf_community` the WORKSPACE name is `org_tensorflow`